image/Dockerfile

```diff
@@ -34,7 +34,7 @@ RUN set -e \
     && rm -rf cypht-temp \
     && apk del .build-deps \
     && cd ${CYPHT_DEST} \
-    && composer install
+    && composer install --ignore-platform-req=ext-session --ignore-platform-req=ext-fileinfo
```

Comment:
This is not exactly safe.  If it's required, you need to put a comment explaining which part of the code or dependency requires it, and explain why it actually works (runs and does the job).